Userland/Libraries/LibJS/Runtime/Temporal/Calendar.cpp

```diff
@@ -299,11 +300,13 @@ Object* to_temporal_calendar(GlobalObject& global_object, Value temporal_calenda
             return &static_cast<PlainDate&>(temporal_calendar_like_object).calendar();
         if (is<PlainDateTime>(temporal_calendar_like_object))
             return &static_cast<PlainDateTime&>(temporal_calendar_like_object).calendar();
+        // TODO: PlainMonthDay
         if (is<PlainTime>(temporal_calendar_like_object))
             return &static_cast<PlainTime&>(temporal_calendar_like_object).calendar();
+        if (is<PlainYearMonth>(temporal_calendar_like_object))
+            return &static_cast<PlainYearMonth&>(temporal_calendar_like_object).calendar();
```

Comment:
Commit message should be `LibJS: Handle PlainYearMonth in ToTemporalCalendar` no?